schemas/test/en/test_list_collector_list_summary.json

```diff
@@ -111,6 +121,10 @@
                     "id": "group",
                     "title": "Questions",
                     "blocks": [
+                        {
+                            "id": "introduction",
+                            "type": "Introduction"
+                        },
```

Comment:
We don't need it?. Preview as a feature can still be accessed?. Non blocker

Reply:
Think it's needed to pass validation?

Reply:
https://github.com/ONSdigital/eq-questionnaire-runner/pull/1151#issuecomment-1617744410